mscore/palettebox.h

```diff
@@ -32,10 +34,13 @@ class PaletteBox : public QDockWidget {
       QVBoxLayout* vbox;
       Palette* newPalette(const QString& name, int slot);
       QComboBox* workspaceList;
-      QLineEdit* searchBox;
+
+      //QMap<QString, Shortcut*> localShortcuts;
+
       const int paletteStretch = 1000;
       QAction* singlePaletteAction;
       QToolButton* addWorkspaceButton;
+      bool keyboardNavigation = false;
```

Comment:
It would be good to add a one line description of this boolean as a comment in the code.